app - Copy.js

```diff
@@ -0,0 +1,52 @@
+var express = require('express');
+var path = require('path');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+var bodyParser = require('body-parser');
+var sequelize = require('sequelize');
+var passport = require('passport');
+var jwt = require('jsonwebtoken');
+
+//Initialize passport strategy
+var hookJWTStrategy = require('./services/passportStrategy');
+
+var app = express();
+
+var indexRouter = require('./routes/index');
+var authRouter = require('./routes/auth');
+var custRouter = require('./routes/customer/profile');
+
+//CORS Middleware
+app.use(function (req, res, next) {
+  //Enabling CORS
+  res.header('Access-Control-Allow-Origin', '*');
```

Comment:
@jmilleriec Does this mean broad allow everything?

Reply:
so that should be gone at this point, if its still in the code, we need to remove it, CORS won't actually work with authn if that's on...